test/cases/eval/process/main.subset.yaml

```diff
@@ -1,24 +1,35 @@
 classMap:
   - type: package
-    name: "."
+    name: .
     children:
       - type: class
         name: main
         children:
           - type: function
             name: f
             location: main.mjs:3
-            static: false
-            source: null
-            labels: []
   - type: package
     name: main.mjs
     children:
       - type: class
+        name: eval-7-2
         children:
           - type: function
             name: g
-            static: false
-            source: null
-            comment: null
-            labels: []
+            location: main.mjs/eval-7-2.js:1
+      - type: class
+        name: eval-7-2
+        children:
+          - type: function
+            name: h
+            location: main.mjs/eval-7-2.js#1:1
```

Comment:
Perhaps the other one could also use the `#0` to mark it as dynamic? I'm not sure about this asymmetry.

Reply:
Good suggestion, done! I did not implemented it at first because I thought it would be challenging. But I found a quick fix.